### ISSUE TITLE
institution_id registration of UCSB

### ISSUE THREAD

**Issue by samanthastevenson** [WCRP-CMIP/CMIP6_CVs#1144]
institution_id = 'UCSB'
institution =  UCSB (University of California, Santa Barbara, USA). Mailing address: Bren School of Environmental Science and Management, c/o Samantha Stevenson, 2400 Bren Hall, University of California Santa Barbara, Santa Barbara, CA 93106, USA

**Comment by chengzhuzhang** [comment]
@durack1 Adding to the issue: We would need to get the institution "UCSB" registered as as a registered institution for the E3SM-1-0 source_id. Thank you!

**Comment by durack1** [comment]
@samanthastevenson @chengzhuzhang, thanks for the update. What `E3SM-1-0` simulations (and under which activity) are planned for `UCSB` submission?

**Comment by samanthastevenson** [comment]
This institution_id is needed for publishing the E3SMv1 Large Ensemble my group at UCSB has been generating! It's part of an RGMA-funded project that I'm a PI on with Matt Newman and Manu Di Lorenzo (lead PI).

**Comment by chengzhuzhang** [comment]
@durack1 Right now we are planning to submit to "CMIP" "historical". And there are also future scenario runs ongoing that I suppose will be submitted to "SenarioMIP" "ssp370" (@samanthastevenson could you confirm if this is planned?)

**Comment by chengzhuzhang** [comment]
@durack1 I do have a question for publishing large ensembles. The E3SM Project has already published 5 historical ensembles. The new runs managed by @samanthastevenson from UCSB has additional 20 ensembles. I'm not sure what is the recommendation for numbering the ensemble variant. The file names for instance (clw_Amon_E3SM-1-0_historical_r1i1p1f1_gr_185001-201412.nc) don't have the 'institution_id' included. if we start with realization 1 (i.e. r1lip1f1) for the UCSB large ensemble, would that cause confusion or not be able to differentiate from the ensembles from E3SM-Project?

**Comment by taylor13** [comment]
There will be confusion only if the model run at UCSB differs from the one that produced the existing 5-member ensemble but has the same source_id.  If the same model is used, then we shouldn't be able to distinguish ensemble members produced by UCSB from the 5 existing members.  That is to say the UCSB realizations belong to the *same* ensemble as the existing members.  The UCSB members should be assigned realization numbers that are unique within the ensemble.  So if the current realizations are numbered 1-5, I would number the UCSB realizations 6-25.

**Comment by durack1** [comment]
@samanthastevenson @chengzhuzhang it would also be good to know if the forcing configuration (and initialization) of these 20 additional ensemble members differ from those already published.

If these differ from the `E3SM-1-0_historical_r1-5i1p1f1` simulations, then it would make sense that the `i` and `f` identifiers are also used to delineate differences in contributed ensemble members.

**Comment by samanthastevenson** [comment]
The model configuration used for the UCSB ensembles is almost, but not quite, identical with that used for the existing 5 DECK simulations. The only differences I'm aware of are a different choice for model timestep and dissipation in MPAS-O, which were implemented by the ocean group after the DECK runs were completed. The relevant namelist parameters are config_dt, config_use_mom_del2, config_mom_del4, and config_btr_dt: in preliminary testing we found that this did lead to a detectable difference in global surface temperature, due most likely to changes in heat absorption into the ocean.

I include the above just for documentation purposes: but given this, we should probably use identifiers that do indicate differences between this ensemble and the previously contributed members. Thanks!

(oh and yes, we do intend to contribute SSP370 members to the ScenarioMIP activity, they're about halfway done at the moment)

**Comment by durack1** [comment]
@samanthastevenson, ok so that means you may want to leverage the `member_id` or `ripf` identifiers to qualify these initialization (`i`), physics (`p`) and forcing (`f`) differences with the existing `r1-5i1p1f1` simulations. The relevant blurb in the [CMIP6 Global Attributes, DRS, Filenames, Directory Structure, and CV’s](https://docs.google.com/document/d/1h0r8RZr_f3-8egBMMh7aqLwy3snpD6_MrDz1q8n5XUk/edit) is reproduced below (Table Note, item 8):

For a given experiment, the realization_index, initialization_index, physics_index, and forcing_index are used to uniquely identify each simulation of an ensemble of runs contributed by a single model.  These indices are defined as follows:
 
realization_index = an integer (≥1) distinguishing among members of an ensemble of simulations that differ only in their initial conditions (e.g., initialized from different points in a control run).  Note that if two different simulations were started from the same initial conditions, the same realization number should be used for both simulations.  For example if a historical run with “natural forcing” only and another historical run that includes anthropogenic forcing were both spawned at the same point in a control run, both should be assigned the same realization.  Also, each so-called RCP (future scenario) simulation should normally be assigned the same realization integer as the historical run from which it was initiated.  This will allow users to easily splice together the appropriate historical and future runs. 
initialization_index = an integer (≥1), which should be assigned a value of 1 except to distinguish simulations performed under the same conditions but with different initialization procedures.  In CMIP6 this index should invariably be assigned the value “1” except for some hindcast and forecast experiments called for by the DCPP activity.  The initialization_index can be used either to distinguish between different algorithms used to impose initial conditions on a forecast or to distinguish between different observational datasets used to initialize a forecast.
physics_index = an integer (≥1) identifying the physics version used by the model.  In the usual case of a single physics version of a model, this argument should normally be assigned the value 1, but it is essential that a consistent assignment of physics_index be used across all simulations performed by a particular model.  Use of  “physics_index” is reserved for closely-related model versions (e.g., as in a “perturbed physics” ensemble) or for the same model run with slightly different parameterizations (e.g., of cloud physics).  Model versions that are substantially different from one another should be given a different source_id” (rather than simply assigning a different value of the physics_index). 
forcing_index = an integer (≥1) used to distinguish runs conforming to the protocol of a single CMIP6 experiment, but with different variants of forcing applied.  One can, for example, distinguish between two historical simulations, one forced with the CMIP6-recommended forcing data sets and another forced by a different dataset, which might yield information about how forcing uncertainty affects the simulation.  

Each data provider can assign whatever positive integers they like  for the realization_index, intialization_index, physics_index, and forcing index.  For each source/experiment pair, however, consistency (in these indices) should be maintained across each parent/child pair whenever sensible (so that, for example, both the ScenarioMIP child and its “historical” parent simulation would be assigned the same set of index values for realization, initialization, and physics); the integer 1 should normally be chosen for each of these in the case of a single variant or for the primary variant (if there is one).  This is only a suggestion, however; there should be no expectation on the part of users that every model will have a value of 1 assigned to any of the r, i, p, f indices, and even if a 1 is assigned it does not imply that it is the primary variant.  Note also that a child spawned by a control run will not  necessarily have the same “ripf” value as the control,  since, for example, multiple realizations of an experiment will  branch from the same control.  

Note that none of the “ripf” indices can be omitted.    

Example of a variant_label:  if realization_index=2, initialization_index=1, physics_index=3, and forcing_index=233, then variant_label = “r2i1p3f233”.

**Comment by taylor13** [comment]
Given that the physics was modified in the model, the UCSB simulations should be treated as a *separate* ensemble from the LLNL-produced simulations.  The important thing is that the "ripf" label should be different for each simulation performed (by a single model for a single experiment).  So if LLNL simulations have labels like r1i1p1f1, r2i1p1f1, r3i1p1f1, ...,  then I suggest you change the "p" value because the physics of the variant is a little different from p1.  You could then assign labels like r1i1p2f1, r2i1p2f1, r3i1p2f1, ...,    
If the method of initializing the model is also different, you could change i1 to i2.  And for any of the indices, you don't have to start with 1 and they don't have to be consecutive.

**Comment by samanthastevenson** [comment]
OK sounds good, thanks for the advice @durack1 and @taylor13! The initialization method is also different from the existing simulations: we analyzed the PI control run for E3SMv1 and chose initial states spanning the phase space of Atlantic and Pacific ocean heat content. So I would think we should indeed probably use the i2 and p2 indices for these ensemble members, and I guess it probably then also would make it more obvious that this is a different ensemble if we started the r indices at 1.
